ios/AirshipFrameworkProxy/AirshipDelegate.swift

```diff
@@ -107,6 +125,16 @@ class AirshipDelegate: NSObject,
 
     }
 
+    func embeddedInfoUpdated(embeddedInfo: [AirshipEmbeddedInfo]) {
```

Comment:
Do you want to add a function to Automation that is `getPendingEmbedddedInfos(embeddedId: String?)`

Reply:
sure